ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -273,12 +274,8 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
                 }),
                 map(defaultAddress => [state, defaultAddress]),
             )),
-            tap(([_, defaultAddress]: [ComponentState, Address | null]) => {
-                if (defaultAddress) {
-                    this.inputAddress$.next(defaultAddress);
-                    this.block.hasValidAddress = true;
```

Comment:
if we don't use hasValidAddress anymore should we refactor MailAddressBlock model?

Reply:
I added it back in a different place. (Though right now this is a redundant validation, but it is convenient).